_im-plugin/ism/policies.md

```diff
@@ -162,6 +163,46 @@ Parameter | Description | Type | Required
 
 For information about setting replicas, see [Primary and replica shards]({{site.url}}{{site.baseurl}}/opensearch#primary-and-replica-shards).
 
+### shrink
+
+Allows you to reduce the number of primary shards in your indexes. With this action, you can specify:
+
+- the number of primary shards that the target index should contain.
+- A max shard size for the primary shards in the target index.
+- Specify a percentage to shrink the number of primary shards in the target index.
+
+```json
+"shrink": {
+    "num_new_shards": {
+        "type": "integer"
+    },
+    "max_shard_size": {
+        "type": "keyword"
+    },
+    "percentage_of_source_shards": {
+        "type": "double"
+    },
+    "target_index_suffix": {
+        "type": "text"
+    },
+    "aliases": {
+        "type": "object"
+    },
+    "force_unsafe": {
+        "type": "boolean"
+    }
+}
```

Comment:
This is a nitpick, but to stay consistent with the rest of the site, we want to indent code blocks with spaces instead of indents. For each level of indentation, I use two spaces.